scripts/perftests/PerformanceTestUtilities.ps1

```diff
@@ -0,0 +1,153 @@
+    function Log([string]$logStatement, [string]$color)
+    {
+        if(-not ([string]::IsNullOrEmpty($color)))
+        {
+            Write-Host "$($(Get-Date).ToString()): $logStatement" -ForegroundColor $color
+        }
+        else
+        { 
+            Write-Host "$($(Get-Date).ToString()): $logStatement"
+        }
+    }
+
+    function GetAbsolutePath([string]$Path)
+    {
+        $Path = [System.IO.Path]::Combine(((pwd).Path), ($Path));
+        $Path = [System.IO.Path]::GetFullPath($Path);
+        return $Path;
+    }
+
+    function OutFileWithCreateFolders([string]$path, [string]$content){
+        $folder = [System.IO.Path]::GetDirectoryName($path)
+        If(!(test-path $folder))
+        {
+            & New-Item -ItemType Directory -Force -Path $folder > $null
```

Comment:
Will this create directories recursively?

Reply:
Yes.